common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/BlockStoreClient.java

```diff
@@ -49,12 +50,24 @@ public abstract void fetchBlocks(
       String host,
       int port,
       String execId,
+      int shuffleGenerationId,
```

Comment:
does it mean this method can only fetch shuffle blocks? Shall we rename the method?

Reply:
Yep, after the refactor e7365e3, finally, we can rename the method and split shuffle fetch and data fetch into 2 separated function, thanks for your guidance and advice!
Done in 67f70a4.